roles/kubernetes/preinstall/templates/ntp.conf.j2

```diff
@@ -0,0 +1,81 @@
+{{ ansible_managed | comment }}
+
```

Comment:
Keep only what you need from upstream samples, don't carry samples that may not be relevant.

Reply:
HI @cristicalin , the file has been clean up. There are only the configure we need.